scripts/community/tradeoffer_new.js

```diff
@@ -35,7 +40,7 @@ if( item !== null )
 	const element = document.createElement( 'script' );
 	element.id = 'steamdb_for_item';
 	element.type = 'text/javascript';
-	element.appendChild( document.createTextNode( '(' + InjectScript.toString() + '(' + item[ 1 ] + ',' + item[ 2 ] + ',' + item[ 3 ] + '))' ) );
+	element.appendChild( document.createTextNode( '(' + InjectScript.toString() + '("' + items.join( ';' ) + '"))' ) );
```

Comment:
This probably should be `JSON.stringify` instead.

Reply:
Now that I think about, the injected script can just read the url, no?

If that works, could you please move the injected script into a separate file so it doesn't need to be stringified, which will align it to how other scripts are injected, see https://github.com/SteamDatabase/BrowserExtension/blob/7fd4f07bca8cff0d1099b34857ae42dbb691889e/manifest.json#L50

Reply:
I thought about it too, but decided not to make too many changes.